DummyTesting/src/main/java/ApiVerification/ApiVerification.java

```diff
@@ -14,6 +14,8 @@ public static void responseCodeValiddation(Response response, int statusCode) {
 		System.out.println("master commit c2");
 
 		System.out.println("new branch commit c3");
+		
```

Comment:
remove the spacing and raise another request

Reply:
spacing removed please review